code/general/struct-vs-hash-vs-openstruct.rb

```diff
@@ -0,0 +1,40 @@
+require "benchmark/ips"
+require "ostruct"
+
+
+Example = Struct.new(:field_1, :field_2)
+Struct.new('User', :field_1, :field_2)
```

Comment:
Yes ! to compare ! 
I saw the second form in the Ruby doc : http://ruby-doc.org/core-2.2.0/Struct.html